CONTRIBUTING.md

```diff
@@ -7,6 +7,10 @@ Contributing to Zeppelin (Source code, Documents, Image, Website) means you agre
 2. If not, create a ticket describing the change you're proposing in the [Jira issue tracker](https://issues.apache.org/jira/browse/ZEPPELIN)
 3. Contribute your patch via Pull Request.
 
+Before you start, please read the [Code of Conduct](http://www.apache.org/foundation/policies/conduct.html) carefully, familiarise yourself with it and refer to it whenever you need it.
+
+For those of who are not familiar with Apache project, understanding how [How it works](http://www.apache.org/foundation/how-it-works.html) would be quite helpful.
```

Comment:
@Leemoonsoo How about deleting "how" ?( i mean the former one )  
Except this, LGTM!